config/packages/security.yaml

```diff
@@ -85,6 +85,9 @@ sulu_security:
         enabled: true
     password_policy:
         enabled: true
+    two_factor:
```

Comment:
would call this `two_factor_authentication`

Reply:
would keep `two_factor` to be consistent to the `scheb/2fa` bundle which is configured above also via `two_factor:`